go-controller/pkg/ovn/master_test.go

```diff
@@ -994,7 +994,16 @@ var _ = ginkgo.Describe("Gateway Init Operations", func() {
 			subnet := ovntest.MustParseIPNet(node1.NodeSubnet)
 			err = clusterController.syncGatewayLogicalNetwork(updatedNode, l3GatewayConfig, []*net.IPNet{subnet}, hostAddrs)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
-
+			clusterController.retryNodes.initRetryObjWithAdd(testNode, testNode.Name)
```

Comment:
could we add a more robust test case that actually forces a node to get retried? There are a couple examples with network policy.

Reply:
I looked at that initally node object unlike others where I can dyn add/delete calling the client apis I will look at this more in the following PRs and see what can possibly done for this resource